.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+const prettier = require('kcd-scripts/prettier');
+
+module.exports = {
+  ...prettier,
+  bracketSpacing: true,
+  semi: true,
```

Comment:
I have a personal preference for double quotes (also the Prettier default). I don't mind much about the rest, i.e. semis etc - can we add it?

```suggestion
  semi: true,
  singleQuote: false,
```